### ISSUE TITLE
9. Revisão da documentação guia para usuário final

### ISSUE THREAD

**Issue by aracele** [pauliceia/issues#3]
**Descrição da tarefa**

<Inserir descrição da tarefa aqui>

**Etapas da tarefa**

- [ ] Inserir etapa + nome pessoa(s) a cargo dela
- [ ] Inserir etapa + nome pessoa(s) a cargo dela
- [ ] Inserir etapa + nome pessoa(s) a cargo dela

**Pessoas envolvidas**

- Vitória + @matchboxxz 
- Inserir nome + @usuario

**Prioridade**
- [ ] 1
- [ ] 2
- [x] 3

**Importante**: Não esquecer de se assinar à tarefa, assim como as demais pessoas envolvidas. Colocar _Labels_ (ex.: documentation) também é importante para a documentação. Usar "fase1.5" como _Milestone_. Fiquem à vontade também para adicionar mais informações que considerarem necessárias ao corpo desta issue.

**Comment by luisferla** [comment]
Sugestão: essa atividade, a exemplo da UX design, precisa contar com a presença de usuários experientes. Além disso, ambas as atividades devem estar concatenadas. Ao menos uma pessoa que participe das duas equipes seria o ideal.

**Comment by matchboxxz** [comment]
Gente não faço ideia do que tem que ser feito nesse ponto, se alguém lembrar ou souber, agradeço a explicação

**Comment by rmmariano** [comment]
@matchboxxz eu **_acho_** que esta issue se refere a revisão dos tutoriais para o usuário final.
Seria isto @luisferla ?

**Comment by luisferla** [comment]
Sim, é isso mesmo, Rodrigo. Por isso, essa atividade está na dependência da atividade UX Design.
